### ISSUE TITLE
Support for Complex numbers

### ISSUE THREAD

**Issue by DoubleStyx** [Xlinka/NeosPlus#77]
Add _Complex_ as a type to Neos in the BaseX namespace.

**Comment by Xlinka** [comment]
when talking about complex numbers do you refer to |2 + 3i| = sqrt(13) as a example of finding the modulus or absolute value of a complex number. ?

**Comment by Xlinka** [comment]
Still no further note on this so no clue what you would like to be added.

**Comment by DoubleStyx** [comment]
Yes, Complex numbers like 1 + 2i, and their associated operations.

https://learn.microsoft.com/en-us/dotnet/api/system.numerics.complex?view=net-7.0

**Comment by Xlinka** [comment]
https://learn.microsoft.com/en-us/dotnet/api/system.numerics.complex?view=netframework-4.6.2

this is the one available to us inside neos

**Comment by Xlinka** [comment]
now with the new injector for types as a base this is now plausable
